cmd/execute.go

```diff
@@ -96,8 +96,8 @@ func (err SummaryTypeError) Error() string {
 var executeCmd = &cobra.Command{
 	Use:     "execute",
 	Aliases: []string{"x"},
-	Short:   "execute exerciser scenario towards a sense installation",
-	Long:    `execute exerciser scenario towards a sense installation`,
+	Short:   "Execute exerciser scenario towards a sense installation",
```

Comment:
Shouldn't this be _gopherciser_ scenario?

Reply:
lol, yea probably, I think you found one of the oldest lines of code in the entire project, probably predates the naming of the repo... ;)

Reply:
should probably also be changed to `a Sense environment` since there's so many different ones now, not only a windows installation.

Reply:
Done